addons/AdvancedTowing/functions/fn_advancedTowingInit.sqf

```diff
@@ -369,13 +369,21 @@ TF47_Attach_Tow_Ropes = {
                 _objDistance = ((_vehicle modelToWorld _vehicleHitch) distance (_cargo modelToWorld _cargoHitch));
                 if( _objDistance > _ropeLength ) then {
                     [["The tow ropes are too short. Move vehicle closer.", false],"TF47_Hint",_player] call TF47_RemoteExec;
-                } else {        
-                    [_vehicle,_player] call TF47_Drop_Tow_Ropes;
+                } else {   
+                    private ["_oldHelper"];
+                    _oldHelper = (_player getVariable ["TF47_Tow_Ropes_Pick_Up_Helper", objNull]);
```

Comment:
bitte nicht mehr private in der Art ``` private ["_oldHelper"]" ``` verwenden, sondern direkt mit der Zuweisung oder initialisierung, aka ``` private _oldHelper = (_player getVariable ["TF47_Tow_Ropes_Pick_Up_Helper", objNull]); ```

Das ist einen Ticken schneller.

Reply:
Ist jetzt egal, aber für die Zukunft, wenn du was neues schreibst.

Reply:
Danke. Done!